feat: improve docs (#1)
stainless-api/readme-action: README.md

# decorate-spec

A GitHub action for generating a "decorated" openapi spec for readme.com. Will take your openapi spec and generate a file in the current working directory called `CUSTOMER-openapi.documented.json`, where `CUSTOMER` is your name.

## Setup

1. Copy the example from below into a GitHub workflow file (e.g. `.github/workflows/decorate.yml`)

> 2. Replace `CUSTOMER` with your company's name

Reviewer: Can we instead use `MY_COMPANY_NAME` as the dummy value instead of `CUSTOMER`?

> 3. Replace `PATH_TO_SPEC` with the relative path to your openapi spec. This path should be relative to the root of the repo.

Reviewer: ```suggestion
3. Replace `PATH_TO_SPEC` with the path to your openapi spec (relative from the root of the repo).
```

4. Add [GitHub actions secrets storing your credentials](https://docs.github.com/en/actions/security-guides/encrypted-secrets)

>    - `secrets.STAINLESS_CONTAINER_TOKEN`: You should've been given a GitHub token, this should be called that.

Reviewer: ```suggestion
   - `secrets.STAINLESS_CONTAINER_TOKEN`: This is the GitHub token we gave you. If you did not receive one, reach out to your contact at Stainless.
```

Reviewer: (can't remember if `gha_` is the prefix)

>    - `secrets.README_TOKEN`: Your API token for README.com.

Reviewer 2: This should probably be `readme.com` right?

>    - `secrets.README_DEFINITION_ID`: According to readme's documentation can be obtained by "clicking edit on the API definition on your project API definitions page".

Reviewer 2: It would be really helpful to link to the relevant part of their documentation here if possible.

>    - `secrets.README_DEFINITION_ID`: According to readme's documentation can be obtained by "clicking edit on the API definition on your project API definitions page".

Reviewer: ```suggestion
   - `secrets.README_DEFINITION_ID`: According to Readme's documentation, this can be obtained by "clicking edit on the API definition on your project API definitions page".
```

Reviewer: Maybe also say a word how this will be used?
